### ISSUE TITLE
vscode 1.77.1 will lost most function with devcontainer and alpine 3.15

### ISSUE THREAD

**Issue by lisbethw1130** [microsoft/vscode#179490]
Does this issue occur when all extensions are disabled?: Yes

- VS Code Version: 1.77.1
- OS Version: MacOS Ventura 13.3（22E252）

Additional information in "About":
```
Version: 1.77.1
Commit: b7886d7461186a5eac768481578c1d7ca80e2d21
Date: 2023-04-04T23:22:38.708Z
Electron: 19.1.11
Chromium: 102.0.5005.196
Node.js: 16.14.2
V8: 10.2.154.26-electron.0
OS: Darwin arm64 22.4.0
Sandboxed: No
```
VSCode will get stuck in v 1.77.1 with dev container and alpine 3.15, and container CPU will keep on 100%, it will become normal if downgrade to 1.76

symptom:

- create new file will stuck in "Running 'File Create' participants..."
- search can only search file which is already opened in vscode, the progress bar keep waiting
- source control panel will always show "scanning folder for git repositories..."

Steps to Reproduce:

1. Create an empty folder
2. open a newly created folder in vscode
3.  Cmd+Shift+P and select "New Dev container"
4. choose "alpine"
5. open devcontainer.json in .devcontainer folder
6. replace image version to 3.15 (in my case, the image name from `mcr.microsoft.com/devcontainers/base:alpine-3.17` to `mcr.microsoft.com/devcontainers/base:alpine-3.15`)
7. rebuild dev container
8. And you will find that most function in vscode stuck or not working(e.g. create new file, search or git), and use htop inside alpine container will see 100% CPU on vscode-server

devcontainer.json
```
// For format details, see https://aka.ms/devcontainer.json. For config options, see the
// README at: https://github.com/devcontainers/templates/tree/main/src/alpine
{
	"name": "Alpine",
	// Or use a Dockerfile or Docker Compose file. More info: https://containers.dev/guide/dockerfile
	"image": "mcr.microsoft.com/devcontainers/base:alpine-3.15"

	// Features to add to the dev container. More info: https://containers.dev/features.
	// "features": {},

	// Use 'forwardPorts' to make a list of ports inside the container available locally.
	// "forwardPorts": [],

	// Use 'postCreateCommand' to run commands after the container is created.
	// "postCreateCommand": "uname -a",

	// Configure tool-specific properties.
	// "customizations": {},

	// Uncomment to connect as root instead. More info: https://aka.ms/dev-containers-non-root.
	// "remoteUser": "root"
}

```

**Comment by Hunterm267** [comment]
I've been experiencing similar issues with VSCode 1.77 within Dev Containers... basic functionality (as mentioned above) as well as extensions and other integrations break. Downgrading to VSCode 1.76 resolves the issues, so presumably this is a Code 1.77 issue of some sort...

**Comment by deepak1556** [comment]
@chrmarti yup, refs https://github.com/microsoft/vscode/issues/178748#issuecomment-1495225249

Merging to https://github.com/microsoft/vscode/issues/178748
